This fixes TLS with WebKit. Caught by clang:

[18/105] Compiling C object tls/openssl/libgioopenssl.so.p/gtlscertificate-openssl.c.o
../tls/openssl/gtlscertificate-openssl.c:151:7: warning: variable 'pkcs8' is used uninitialized whenever 'if' condition is true [-Wsometimes-uninitialized]
  if (!openssl->key)
      ^~~~~~~~~~~~~
../tls/openssl/gtlscertificate-openssl.c:176:7: note: uninitialized use occurs here
  if (pkcs8)
      ^~~~~
../tls/openssl/gtlscertificate-openssl.c:151:3: note: remove the 'if' if its condition is always false
  if (!openssl->key)
  ^~~~~~~~~~~~~~~~~~
../tls/openssl/gtlscertificate-openssl.c:147:29: note: initialize the variable 'pkcs8' to silence this warning
  PKCS8_PRIV_KEY_INFO *pkcs8;
                            ^
                             = NULL
1 warning generated.

--- a/tls/openssl/gtlscertificate-openssl.c
+++ b/tls/openssl/gtlscertificate-openssl.c
@@ -144,7 +144,7 @@ export_privkey_to_der (GTlsCertificateOpenssl  *openssl,
                        guint8                 **output_data,
                        long                    *output_size)
 {
-  PKCS8_PRIV_KEY_INFO *pkcs8;
+  PKCS8_PRIV_KEY_INFO *pkcs8 = NULL;
   BIO *bio = NULL;
   const guint8 *data;
 
